src/AddTodoForm.jsx

```diff
@@ -0,0 +1,14 @@
+const AddTodoForm = () => {
+    return (
+        <>
```

Comment:
Consider simplifying the component by removing the unnecessary fragment to make the code cleaner - as the `form` tag itself acts as a container.